tests/stubs/config.js

```diff
@@ -38,6 +38,14 @@ module.exports = {
       'secret': 'AWS_S3_SECRET',
       'region': 'AWS_S3_REGION',
       'bucket': 'AWS_S3_BUCKET'
+    },
+
+    'ftp': {
+      'driver': 'ftp',
+      'host': 'FTP_HOST',
+      'port': 21,
+      'user': 'FTP_USER',
+      'pass': 'FTP_PASS'
```

Comment:
Could be great to add an `longLive` that keep the connection open if we have many actions to do and to close it by default after one action.